example/src/examples/cow_repository.dart

```diff
@@ -97,17 +97,19 @@ class CopyAndWriteRepository extends PackageRepository {
   bool get supportsUpload => true;
 
   @override
-  Future<PackageVersion> upload(Stream<List<int>> data) {
+  Future<PackageVersion> upload(Stream<List<int>> data) async {
     _logger.info('Starting upload to local package repository.');
     // TODO: Converting this to an async scope makes the stream not get any data
     // or done event. Seems like there is still an issue in
     // package:mime - making this an async scope results in this stream getting
     // no data.
```

Comment:
This comment then no longer applies. Mmy change in https://github.com/dart-lang/mime/commit/a469a155545b17b27f122e4069ddedbef3a364fc or Soeren's change https://github.com/dart-lang/mime/commit/aba18786be251ef15ac59248795a7399d0491d0f fixed it most likely.

Reply:
ack